packages/js/product-editor/src/components/block-editor/block-editor.tsx

```diff
@@ -198,8 +198,12 @@ export function BlockEditor( {
 		product
 	);
 
+	const isProductLoaded = product && Object.keys( product ).length > 0;
```

Comment:
Because the API returns an empty object while it's loading... I really wish it returned `null` instead...

Reply:
Ah, got it. So, thinking a bit more, since this check of the keys doesn't really feel right... I think we should use the `isResolving` value returned from `useEntityRecord` to handle this.

https://developer.wordpress.org/block-editor/reference-guides/packages/packages-core-data/#useentityrecord